### PR TITLE
New package: lapce-0.2.0

### PR DESCRIPTION
**Pull request by simvux**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/lapce/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'lapce'
+pkgname=lapce
+version=0.2.0
+revision=1
+build_style=cargo
+hostmakedepends="cargo python cmake pkg-config"
```
Comment (classabbyamp) [inline srcpkgs/lapce/template:6]: does this work with python3? new python 2 packages will not be accepted

File: srcpkgs/lapce/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+makedepends="expat fontconfig-devel freetype gtk+3-devel libxcb openssl-devel glib-devel cairo-devel atk-devel gdk-pixbuf-devel"
+short_desc="Lighting-fast and powerful code editor"
+maintainer="Simon <simon.larsson-general@protonmail.com>"
+license="Apache-2.0"
+homepage="https://lapce.dev"
+distfiles="https://github.com/${pkgname}/${pkgname}/archive/v${version}/$pkgname-$pkgver.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/lapce/template:12]: what is pkgver supposed to be?

Comment (simvux) [reply srcpkgs/lapce/template:12]: oops, supposed to be `version` but I'm surprised it still worked. Seems like github allows any filename as long as it ends with `.tar.gz` and it'll redirect to the right one

File: srcpkgs/lapce/template (hunk 1 above)
Comment (simvux) [reply srcpkgs/lapce/template:6]: The official rpm package builds without python at all (even though the arch package has it as dependency), so I'll give that a go.

**Comment by hayley** [comment]
Thank you for this @simvux! By looking at your `makedepends`, I was able to figure out what I needed to install to get lapce to build.

**Comment by simvux** [comment]
Closing in favor of #40970 
Unsure why this PR was completely forgotten and abandoned ):

**Pull request closed without merging**